### DISCUSSION TITLE
v0.1

### DISCUSSION THREAD

**Discussion by ryanlong1004** [esmf-org/esmf-profiler#52] [Announcements]
Pre-release version.

Please submit any issues to https://github.com/esmf-org/esmf-profiler/issues.

## What's Changed
* Create python-app.yml by @ryanlong1004 in https://github.com/esmf-org/esmf-profiler/pull/1
* Web gui by @ryanlong1004 in https://github.com/esmf-org/esmf-profiler/pull/9
* Add site name and timestamp as headers in Report. by @ryanlong1004 in https://github.com/esmf-org/esmf-profiler/pull/24
* Center select all and select none buttons. by @ryanlong1004 in https://github.com/esmf-org/esmf-profiler/pull/23
* Changed destination from esmf site to '#' to prevent redirection. by @ryanlong1004 in https://github.com/esmf-org/esmf-profiler/pull/25
* removed conditional rendering (not-redrawing) of chart. by @ryanlong1004 in https://github.com/esmf-org/esmf-profiler/pull/26
* Closes #18 by @ryanlong1004 in https://github.com/esmf-org/esmf-profiler/pull/27
* Add version compare function that accounts for different version lengths by @ryanlong1004 in https://github.com/esmf-org/esmf-profiler/pull/33
* Fixes #38 static web files issue by @ryanlong1004 in https://github.com/esmf-org/esmf-profiler/pull/40
* closes #29 container implementation by @ryanlong1004 in https://github.com/esmf-org/esmf-profiler/pull/43
* 41 refactor main analysis by @ryanlong1004 in https://github.com/esmf-org/esmf-profiler/pull/44
* 46 testing by @ryanlong1004 in https://github.com/esmf-org/esmf-profiler/pull/47
* Development to Main for Pre-Release v0.1 by @ryanlong1004 in https://github.com/esmf-org/esmf-profiler/pull/50

## New Contributors
* @ryanlong1004 made their first contribution in https://github.com/esmf-org/esmf-profiler/pull/1

**Full Changelog**: https://github.com/esmf-org/esmf-profiler/commits/v0.1

<hr /><em>This discussion was created from the release <a href='https://github.com/esmf-org/esmf-profiler/releases/tag/v0.1'>v0.1</a>.</em>

**Comment by rsdunlapiv** [comment]
@ryanlong1004 congrats!  Great to see version 0.1 released!
